### ISSUE TITLE
Missing Fields

### ISSUE THREAD

**Issue by amit-trexo** [parkerhancock/patent_client#158]
It seems we are now getting fewer fields , I do not see attorney details + parent child continuity etc.
Please check, we are now getting the confirmation number, but missing the other details which were coming in V4.1.10

**Comment by parkerhancock** [comment]
Sadly, that's not on me - that's a USPTO issue. The [PEDS API](https://ped.uspto.gov/peds/#/) that powers USApplication will be deprecated by the end of the year in favor of the USPTO's new [Open Data Portal](https://beta-data.uspto.gov/home). 

In the mean time, I've noticed that PEDS is no longer providing attorney details, file history information, continuity information, and others. I suspect that means it is already no longer being maintained, and as things break, they are just taking them offline rather than fixing them.

The solution is to start using the new ODP API, documented [here](https://patent-client.readthedocs.io/en/latest/user_guide/open_data_portal.html). It has all the fields you're looking for, but may be a bit buggy (please report them!)

Just replace this:

```python
from patent_client import USApplication
```

with this:

```python
from patent_client.odp import USApplication
```

Thanks!

**Comment by amit-trexo** [comment]
Thanks !  But when i downgrade it to 4.1.10. It is giving me all the fields and hence it should not be PEDS. So , I am now using the old version instead of the latest.

**Comment by parkerhancock** [comment]
Weird! It's back in the client app now too!

In any case, I'd treat PEDS with some suspicion, and migrate to ODP as quickly as is reasonable.

Thanks,

**Comment by amit-trexo** [comment]
Since ODP is still in beta and not declared live. Is it possible to get all the missing fields back in new version (V5).

**Comment by parkerhancock** [comment]
Yes! I'll add them back in a version. Should be able to get to that this afternoon.

**Comment by parkerhancock** [comment]
Fixed in v5.0.12

Thanks!
